test/bigdecimal/test_bigmath.rb

```diff
@@ -6,10 +6,30 @@ class TestBigMath < Test::Unit::TestCase
   include TestBigDecimalBase
   include BigMath
   N = 20
+  N_LARGE = 100
```

Comment:
This is just a comment to share my concern regarding the use of BigDecimal on 64-bit architectures. Each BigDecimal digit element can hold 19 decimal digits, meaning that 100 would require 6 BigDecimal digits. We should consider if 6 BigDecimal digits is truly a large number.

Reply:
Update the precision assertion to use [50, 100, 150]
Trying multiple precisions will increase the probability of detecting mis-calculation.